edweiss-app/components/input/FancyButton.tsx

```diff
@@ -16,6 +16,8 @@ type FancyButtonProps = MarginProps & Testable & {
 	style?: ContainerStyle,
 	icon?: IconType,
 	outlined?: boolean;
+	radius?: Size;
+	px?: Size;
 
 	onPress?(): void,
 	children?: ReactNode,
```

Comment:
Why do you use `?` 
Maybe you can add some defaults attributs.

Reply:
I prefer to leave it like this since outlined, style etc... don't have default values. It does not poses any problem to leave it like this imo.